config.py

```diff
@@ -1470,6 +1470,18 @@ def get_t1_from_meeg(bids_path):
     ```
 """
 
+source_info_update: Optional[Dict[str, str]] = None
```

Comment:
if you make it default to `dict(suffix='ave')` then if you avoid the if and you can document that 
is this case you take the info from the evoked by default

Reply:
that makes sense.